tests/server_tests.py

```diff
@@ -162,10 +162,12 @@ def __init__(self, port, smtp_port):
 
     def flush_output(self):
       """Flushes the buffered output from this subprocess to stderr."""
-      self.output, lines_to_print = [], self.output
-      if lines_to_print:
-        if self.BIND_RE.search('\n'.join(lines_to_print)):
-            sys.stderr.write('%s failed port 8081 already in use.\n' % self.name)
+      self.output, original_output = [], self.output
+      if original_output:
+          matchOB = self.BIND_RE.search('\n'.join(original_output), re.MULTILINE);
+          if matchOB:
```

Comment:
If this doesn't match (i.e., it fails with some other reason), shouldn't we output the all original_output as we originally did?